modules/social_features/social_group/social_group.module

```diff
@@ -2807,11 +2807,12 @@ function _social_group_action_batch_finish($success, array $results, array $oper
     return;
   }
 
-  $operations = array_count_values($results['operations']);
-  $results_count = 0;
-
-  foreach ($operations as $count) {
-    $results_count += $count;
+  // Count changed entities
```

Comment:
Please fix CS here and other places: **Inline comments must end in full-stops.**

Reply:
Done